IonicPortals.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.source = { :git => 'https://github.com/ionic-team/ionic-portals-ios.git', :tag => s.version.to_s }
   s.source_files = 'IonicPortals/IonicPortals/*.{swift,h,m}'
   s.resources = ['IonicPortals/IonicPortals/**/*.{xib,xcassets,svg}']
-  s.dependency 'Capacitor'
-  s.dependency 'IonicLiveUpdates'
+  s.dependency 'Capacitor', '~> 3.5'
+  s.dependency 'IonicLiveUpdates', '~ 0.1.0'
```

Comment:
whoops, need to make '~ 0.1.0' be '~> 0.1.0'. Should probably add a `pod spec lint` to the CI cause this wasn't caught by it